docs/docs/add-a-manifest-file.md

```diff
@@ -53,4 +52,4 @@ npm install --save gatsby-plugin-manifest
 }
 ```
 
-That's all you need to get started with adding a web manifest to a Gatsby site. The example given reflects a base configuration -- check out the [plugin reference](/packages/gatsby-plugin-manifest/?=gatsby-plugin-manifest#automatic-mode) for more options.
+Itu saja yang kamu butuhkan untuk menambahkan file manifest di website Gatsby. Contoh yang diberikan menggambarkan konfigurasi basis -- lebih lanjut lihat referensi di [plugin reference](/packages/gatsby-plugin-manifest/?=gatsby-plugin-manifest#automatic-mode) untuk opsi lebih lanjut.
```

Comment:
For “plugin reference”, please change it to “referensi _plugin_”

Reply:
Thanks for the review 🙏